lnrpc/rpc.proto

```diff
@@ -819,7 +819,7 @@ service Lightning {
 
 message Utxo {
     /// The type of address
-    AddressType type = 1;
+    AddressType address_type = 1;
```

Comment:
note: usually it is forbidden to change name for existing fields. We may choose to do this if our users use submitted generated code and json, otherwise I'd reconsider.

Reply:
Yes, we are aware of that. See the discussion in #3996.